packages/expect/src/asymmetricMatchers.ts

```diff
@@ -146,14 +145,15 @@ class ArrayContaining extends AsymmetricMatcher {
 
 class ObjectContaining extends AsymmetricMatcher {
   sample: Object;
+  inverse: boolean;
 
   constructor(sample: Object, inverse: boolean = false) {
     super();
     this.sample = sample;
     this.inverse = inverse;
   }
 
-  asymmetricMatch(other: Object) {
+  asymmetricMatch(other: any) {
```

Comment:
does `unknown` work?

Reply:
This doesn't work without another type check. The compiler complains `Object is of type 'unknown'`